math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -642,21 +642,19 @@ RVec<T> Filter(const RVec<T> &v, F &&f)
 }
 
 /// Return true if any of the elements equates to true, return false otherwise.
-template <typename T>
-auto Any(const RVec<T> &v) -> decltype(v[0] == true)
+inline bool Any(const RVec<RStrongBool> &v)
```

Comment:
it's a bit weird that the following shouldn't compile:
```c++
RVec<bool> vals{true,false};
Any(vals);
```

Maybe `RVec<bool>` should be implicitly convertible to `RVec<RStrongBool>`? Or is it too messy?